vita3k/ctrl/src/ctrl.cpp

```diff
@@ -282,10 +282,10 @@ static void retrieve_ctrl_data(EmuEnvState &emuenv, int port, bool is_v2, bool n
         }
     };
 
-    if (emuenv.common_dialog.status == SCE_COMMON_DIALOG_STATUS_RUNNING) {
+    if (emuenv.common_dialog.status == SCE_COMMON_DIALOG_STATUS_RUNNING || emuenv.ctrl.drop_inputs) {
         if (negative)
             buttons ^= ~0;
-        reset_axes();
+        reset_axes(); // keep sticks centered
```

Comment:
Remove comment, is not needed, function name already explain what it done

Reply:
Quite frankly, it was not obvious to me, someone who rarely touches the code. Documentation is helpful.

Reply:
Just remove 🙏 is full clear

Reply:
![image](https://github.com/Vita3K/Vita3K/assets/39563889/c749a739-a073-4f58-9e47-49b202f0d6d7)

Reply:
Th8s  ode is not me, i have hyst create function, so yes number is magic, but function of name is clear

Reply:
name of function is not clear. if you renamed it to, say, `recenter_joysticks`, it would be clear.

Reply:
Here you only adding comment in one time, no have sense, if you want, adding comment inside lambda function, but is already clear

Reply:
i'll do that then.

Reply:
> name of function is not clear. if you renamed it to, say, `recenter_joysticks`, it would be clear.

Name reset axis is clear
If tiu nocknow what is axis 🤷‍♂️